public/Overture_Maps_Example/Overture_Maps_Example.py

```diff
@@ -77,12 +94,16 @@ def get_part(part):
 
     if len(dfs):
         df = pd.concat(dfs)
-        print(df.columns)
-        for col in df.columns:
-            # Some overture columns do not serialize nicely and can have compatability
-            # issues with some Parquet implementations.
-            # Here we coerce to string to work around that.
-            if col != "geometry":
-                df[col] = df[col].apply(str)
-        return df
-    return None
+        # Some overture columns do not serialize nicely and can have compatability
+        # issues with some Parquet implementations.
+        # Here we coerce to string to work around that.
+        gdf = gpd.GeoDataFrame(pd.concat([df[[c for c in df.columns if c != "geometry"]].astype(str), df[["geometry"]]], axis=1))
```

Comment:
May I suggest introducing the category column? Because the `categories` json gets stringified into a non-valid json (due to the array object), it's cumbersome to extract the `category` value from the response.

```suggestion
        gdf = gpd.GeoDataFrame(pd.concat([df[[c for c in df.columns if c != "geometry"]].astype(str), df[["geometry"]]], axis=1))
        df['category'] = df['categories'].apply(lambda x: x['main'] if x else -1)
```

Reply:
Really trying to avoid using “apply” as it can slow things down when there’s a million features.

What’s the desired data and do you have an example?